packages/wordpress-external-dependencies-plugin/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@automattic/wordpress-external-dependencies-plugin",
+  "version": "0.1.0",
+  "description": "Webpack plugin to externalize WordPress dependencies",
+  "keywords": [
+    "WordPress",
+    "Webpack"
+  ],
+  "author": "Jon Surrell <jon.surrell@gmail.com>",
+  "homepage": "https://github.com/Automattic/wp-calypso/packages/webpack-plugin-extern-wp-deps",
+  "license": "GPL-2.0-or-later",
+  "main": "index.js",
+  "sideEffects": false,
```

Comment:
This is a webpack plugin that's ever run only from Node.js, no browser or bundler is ever involved. The`sideEffects` flag is not needed and `main` has default value anyway.

Reply:
Should docs then be updated? It says:

> Your package.json can have any of the normal properties but at a minimum should contain `main`, `module`, and `sideEffects`.

https://github.com/Automattic/wp-calypso/blob/2b2ca2b01466b33add52c533cb6383cef7080be2/docs/monorepo.md